tests/test_runs/test_run_functions.py

```diff
@@ -701,9 +715,11 @@ def test_initialize_cv_from_run(self):
             cv=StratifiedKFold(n_splits=2, shuffle=True),
             n_iter=2)
 
+        clf = make_pipeline(OneHotEncoder(handle_unknown='ignore'), randomsearch)
```

Comment:
Can you please put the one hot encoder inside a pipeline that's inside random search?

Reply:
I am not sure if I understood the requirement completely. However, made a change based on how I understood it.
Kindly let me know if this is not what was meant.

Reply:
Yes, that's exactly how it should look.